OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -19,6 +19,11 @@
     <Configurations>Release;Debug;Release-x86</Configurations>
     <ManagedExeLauncher Condition="('$(OS)' != 'Windows_NT')">mono </ManagedExeLauncher>
```

Comment:
I'm not sure if we can/want to merge these overrides in with the netfx.props definitions?